### ISSUE TITLE
When renaming a label, there should be the option of using a dialog box

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#1909]
When renaming a selected clip using the keyboard, if the "Use dialog for the name of new label" option is checked, then a dialog is opened for entering the name of the clip. This enables both users of screen readers and IME to rename the clip. The custom text box normally used is not accessible for users of screen readers, and doesn't work with an IME.

It would be very helpful if there the same option when renaming a label in a label track.

**Comment by LWinterberg** [comment]
Hello! We just [switched development to Audacity 4](https://forum.audacityteam.org/t/master-has-been-switched-to-4-0-0-development/120444). We've redone, or plan on redoing, various things from scratch, including the entire UI, and the part this bug is about. As such, I'm closing this bug as it will be fixed there and we won't be fixing it for the final releases of Audacity 3.
